### PR TITLE
ref(tags-table): Replace getMeta with _meta object - [TET-144]

### PR DESCRIPTION
**Pull request by priscilawebdev**
**What this PR does:**

- Replace `getMeta` function in the tags table with the `_meta` object
- Add a test validating the meta
- Update the logic to display a context - it shall only display an empty value when a scrubbed rule of that property is found in the **_meta** object

### CODE REVIEW

File: static/app/utils/analytics/samplingAnalyticsEvents.tsx [hunk 1]
```diff
@@ -8,6 +8,9 @@ type Rule = {
 };
 
 export type SamplingEventParameters = {
+  'sampling.sdk.client.rate.change.alert': {
+    project_id: string;
+  };
```
Comment (matejminar) [inline static/app/utils/analytics/samplingAnalyticsEvents.tsx:13]: These changes look unrelated. Probably leftovers from a different branch?

File: static/app/components/events/contexts/app/getAppKnownData.tsx [hunk 2]
```diff
@@ -16,11 +15,12 @@ export function getAppKnownData({data, event, meta}: Props): KeyValueListData {
...
   const dataKeys = appKnownDataValues.filter(appKnownDataValue => {
-    if (!defined(data[appKnownDataValue])) {
-      if (meta[appKnownDataValue]) {
-        return true;
-      }
-      return false;
+    if (
+      typeof data[appKnownDataValue] !== 'number' &&
+      typeof data[appKnownDataValue] !== 'boolean' &&
+      !data[appKnownDataValue]
```
Comment (matejminar) [inline static/app/components/events/contexts/app/getAppKnownData.tsx:21]: We are repeating this check quite a lot, should we create a helper for it?

File: static/app/utils/analytics/samplingAnalyticsEvents.tsx (hunk 1 above)
Comment (priscilawebdev) [reply static/app/utils/analytics/samplingAnalyticsEvents.tsx:13]: oh no... let me remove it... thanks

**Review by matejminar — APPROVED** [review]
